modules/storage-ephemeral-storage-manage.adoc

```diff
@@ -40,17 +58,33 @@ spec:
     image: images.my-company.example/log-aggregator:v6
     resources:
       requests:
-        ephemeral-storage: "2Gi" <1>
+        ephemeral-storage: "2Gi"
+      limits:
+        ephemeral-storage: "4Gi"
     volumeMounts:
     - name: ephemeral
       mountPath: "/tmp"
   volumes:
     - name: ephemeral
       emptyDir: {}
 ----
-<1> Request for local ephemeral storage.
-<2> Limit for local ephemeral storage.
+<1> Container request for local ephemeral storage.
+<2> Container limit for local ephemeral storage.
```

Comment:
Just for comparative purposes with the comment on line 63, this <2> call out text appears both here and in the preview.

Reply:
These are on the previous two lines, 51 and 53. I removed the second set. Looks OK now, so I wonder if everything on the GitHub side just didn't update.